frontend/ui/src/pages/Inbox/MessageInput/index.tsx

```diff
@@ -36,10 +36,20 @@ const MessageInput = (props: MessageInputProps & ConnectedProps<typeof connector
   const conversationIdParams = useParams();
   const currentConversationId: string = conversationIdParams[Object.keys(conversationIdParams)[0]];
 
-  const handleClick = () => {
+  const sendMessage = () => {
     props.sendMessages(getTextMessagePayload(channelSource, currentConversationId, input)).then(() => setInput(''));
   };
 
+  const handleClick = () => {
```

Comment:
nitpick: What's the point of the extra wrap? :)

Reply:
Could have called the method down below, but we will add more stuff to it soon. Would have to introduce the method again.